kano_world/hooks/launch.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
```

Comment:
Indeed it doesn't. Blame my too broad copy-paste skills, I will remove it from the other file as well